src/constants/COLOR_RELATIONSHIP.js

```diff
@@ -0,0 +1,20 @@
+const COLOR_RELATIONSHIP = {
+  지인: {
+    bgColor: "bg-beige-100",
+    textColor: "text-beige-500",
+  },
```

Comment:
bgColor, textColor가 나눠서 재사용될 일이 없으면 한꺼번에 작성하는 방법도 괜찮을 것 같습니다.

```suggestion
  지인: "bg-beige-100 text-beige-500",
```

Reply:
좋은 것 같습니다 감사합니다